src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/services/mapper/CaseDataBuilderTest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.reform.sscs.services.mapper;
+
+import static com.google.common.collect.Lists.newArrayList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sscs.models.GapsEvent;
+import uk.gov.hmcts.reform.sscs.models.deserialize.gaps2.AppealCase;
+import uk.gov.hmcts.reform.sscs.models.deserialize.gaps2.Hearing;
+import uk.gov.hmcts.reform.sscs.models.deserialize.gaps2.MajorStatus;
+import uk.gov.hmcts.reform.sscs.models.deserialize.gaps2.MinorStatus;
+import uk.gov.hmcts.reform.sscs.models.serialize.ccd.Event;
+import uk.gov.hmcts.reform.sscs.models.serialize.ccd.Events;
+import uk.gov.hmcts.reform.sscs.services.sftp.SftpChannelAdapter;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class CaseDataBuilderTest {
+
+    private static final String TEST_DATE2 = "2018-05-24T00:00:00+01:00";
+    private static final String TEST_DATE = "2017-05-24T00:00:00+01:00";
```

Comment:
Use meaningful variable name instead of just TEST_DATE. Its actually datetime value.

Reply:
these test dates are used indistinctively either in minor status or major status. I think the name represents well the intention which is just a couple of test dates to be used in the minor status or major status.